src/include/elektra.h

```diff
@@ -0,0 +1,368 @@
+/**
+ * @file
+ *
+ * @brief Elektra High Level API.
+ *
+ * @copyright BSD License (see doc/LICENSE.md or http://www.libelektra.org)
+ */
+
+#ifndef ELEKTRA_H
+#define ELEKTRA_H
+
+#include "elektra_error.h"
```

Comment:
I think a directory would be the better solution then. Otherwise some of the names would become quite illegible (e.g. the generated `elektraerrorcodes.h`).

Reply:
Yes, directory is the best way forward. @waht already used this for kdbio and the result is very elegant.